taqueria-vscode-extension/package.json

```diff
@@ -106,64 +106,90 @@
-					"when": "@taqueria-state/is-taq-cli-reachable && !@taqueria-state/enable-init-scaffold"
+					"when": "@taqueria-state/system-check-passed && !@taqueria-state/enable-init-scaffold"
 				},
 				{
 					"id": "taqueria-contracts",
 					"name": "Contracts",
 					"contextualTitle": "Taqueria: Contracts",
-					"when": "@taqueria-state/is-taq-cli-reachable && !@taqueria-state/enable-init-scaffold"
+					"when": "@taqueria-state/system-check-passed && !@taqueria-state/enable-init-scaffold"
 				},
 				{
 					"id": "taqueria-sandboxes",
 					"name": "Sandboxes",
 					"contextualTitle": "Taqueria: Sandboxes",
-					"when": "@taqueria-state/is-taq-cli-reachable && !@taqueria-state/enable-init-scaffold"
+					"when": "@taqueria-state/system-check-passed && !@taqueria-state/enable-init-scaffold"
 				},
 				{
 					"id": "taqueria-artifacts",
 					"name": "Artifacts",
-					"contextualTitle": "Artifacts"
+					"contextualTitle": "Taqueria: Artifacts",
+					"when": "@taqueria-state/system-check-passed && !@taqueria-state/enable-init-scaffold"
 				},
 				{
 					"id": "taqueria-environments",
 					"name": "Environments",
 					"contextualTitle": "Taqueria: Environments",
-					"when": "@taqueria-state/is-taq-cli-reachable && !@taqueria-state/enable-init-scaffold"
+					"when": "@taqueria-state/system-check-passed && !@taqueria-state/enable-init-scaffold"
 				},
 				{
 					"id": "taqueria-tests",
 					"name": "Tests",
 					"contextualTitle": "Taqueria: Tests",
-					"when": "@taqueria-state/is-taq-cli-reachable && !@taqueria-state/enable-init-scaffold"
+					"when": "@taqueria-state/system-check-passed && !@taqueria-state/enable-init-scaffold"
 				}
 			]
 		},
 		"viewsWelcome": [
 			{
-				"view": "taqueria-cli-check",
-				"contents": "Unable to find your Taqueria cli (taq command).\nPlease follow instructions [here](https://taqueria.io/docs/getting-started/installation/) to install it, then click the button below:\n[Check again](command:taqueria.refresh_command_states)\nAlso, please make sure it is in your PATH.\nIf you change your PATH, you might need to restart VsCode to pick up the change."
+				"view": "taqueria-system-check",
+				"when": "@taqueria-state/is-taq-cli-reachable",
+				"contents": "✅ Taq CLI: Installed"
+			},
+			{
+				"view": "taqueria-system-check",
+				"when": "!@taqueria-state/is-taq-cli-reachable",
+				"contents": "❌ Taq CLI: Unable to find the Taqueria CLI.\nThe Taqueria CLI is required to use this extension. Click [here](https://taqueria.io/docs/getting-started/installation/) for installation instructions, then click the \"Check Again\" button.\nThe Taqueria CLI must also be in your PATH. If you change your PATH, you might need to restart Visual Studio Code to register this change."
+			},
+			{
+				"view": "taqueria-system-check",
+				"when": "@taqueria-state/is-node-installed && @taqueria-state/node-version-meets-requirements",
+				"contents": "✅ NodeJs: Installed"
```

Comment:
`NodeJs` ideally should be `Node.js`

If a period in a line is going to cause a headache, the former is fine though

Reply:
Done. Thanks.